docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -312,7 +312,7 @@ compatible because they have a different ingestion spec or partition allocation,
 supervisor will create a new set of tasks. In this way, the supervisors are persistent across overlord restarts and
 fail-overs.
 
-A supervisor is stopped via the `POST /druid/indexer/v1/supervisor/<supervisorId>/shutdown` endpoint. This places a
+A supervisor is stopped via the `POST /druid/indexer/v1/supervisor/<supervisorId>/terminate` endpoint. This places a
```

Comment:
why do we need this backward incompatible change or this is fixing the doc?

Reply:
`/shutdown` and `/terminate` are equivalent since #6272, although `/shutdown` is deprecated. (On the grounds that the difference between 'suspend' and 'terminate' is more clear than 'suspend' and 'shutdown'.)

So this was just fixing the doc.